### PR TITLE
Add PodFetch

### PR DESCRIPTION
**Pull request by DrMxrcy**

### CODE REVIEW

**Comment by DrMxrcy** [comment]
Works, just buggy at the moment. Waiting on some bugs to be released.

File: apps/podfetch/config.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "max": 50,
+      "min": 3,
+      "required": true,
+      "env_variable": "PODFETCH_USERNAME"
+    },
+    {
+      "type": "password",
+      "label": "PodFetch Password",
+      "max": 50,
+      "min": 3,
```
Comment (meienberger) [inline apps/podfetch/config.json:28]: I think you can increase this to 8, I'm afraid in a future update they require a longer password and thus break existing installations

File: apps/podfetch/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - ${APP_PORT}:8000
+    volumes:
+      - ${APP_DATA_DIR}/data/podfetch-db:/app/db
+      - ${ROOT_FOLDER_HOST}/media/data/podcasts:/app/podcasts
+    environment:
+      - BASIC_AUTH=true
+      - USERNAME=${PODFETCH_USERNAME}
+      - PASSWORD=${PODFETCH_PASSWORD}
+      - POLLING_INTERVAL=60
+      - SERVER_URL=https://${APP_DOMAIN}
```
Comment (meienberger) [inline apps/podfetch/docker-compose.yml:17]:
```suggestion
      - SERVER_URL=${APP_PROTOCOL:-http}://${APP_DOMAIN}
```

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/podfetch/config.json (hunk 1 above)
Comment (DrMxrcy) [reply apps/podfetch/config.json]: Resolved in 7e5dfa0

File: apps/podfetch/docker-compose.yml (hunk 2 above)
Comment (DrMxrcy) [reply apps/podfetch/docker-compose.yml]: Resolved in 7e5dfa0

**Review by meienberger — APPROVED** [review]
+1 📄

**Pull request merged**